ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplLoadTask.java

```diff
@@ -298,25 +298,26 @@ a database ( directory )
    * @throws IOException File operations failure.
    * @throws InvalidInputException Invalid input dump directory.
    */
-  private void bootstrapRollbackTask() throws HiveException, IOException, InvalidInputException {
-    Path bootstrapDirectory = new PathBuilder(work.bootstrapDumpToRollback)
+  private void cleanTablesFromBootstrap() throws HiveException, IOException, InvalidInputException {
+    Path bootstrapDirectory = new PathBuilder(work.bootstrapDumpToCleanTables)
             .addDescendant(ReplUtils.INC_BOOTSTRAP_ROOT_DIR_NAME).build();
     FileSystem fs = bootstrapDirectory.getFileSystem(conf);
 
     if (!fs.exists(bootstrapDirectory)) {
-      throw new InvalidInputException("Input bootstrap dump directory to rollback doesn't exist: "
+      throw new InvalidInputException("Input bootstrap dump directory to clean tables is invalid: "
```

Comment:
Grammar suggestion "Bootstrap dump directory specified to clean tables from is invalid"

Reply:
Done